apierror.h

```diff
@@ -68,6 +70,6 @@
 /*! \brief Helper method to get a string representation of an API error code
  * @param[in] error The API error code
  * @returns A string representation of the error code */
-const char *janus_get_api_error(int error);
+shared const char *janus_get_api_error(int error);
```

Comment:
Yep but as Saul pointed out, I'd use a different name for that. `shared` is too generic and risks to cause conflicts. Better something that explicitly belongs to the Janus namespace, e.g., `JANUS_EXTERN` or `JANUS_API` as Saul suggested, or something like that. Open to suggestions!